### PR TITLE
add `MAVEN_REPO` to AccessChannel

### PR DESCRIPTION
**Pull request by hjmodi**

### CODE REVIEW

File: src/main/java/org/commonjava/indy/service/tracking/model/AccessChannel.java [hunk 1]
```diff
@@ -29,4 +29,9 @@
     /** Used to signify content coming from normal repositories and groups. */
     NATIVE,
 
+    /** Used when the store is accessed via regular Maven repo.
+     *  NOTE: This has been changed to {@link #NATIVE} in our tracking code. It is included for historical purposes. */
+    @Deprecated
+    MAVEN_REPO;
```
Comment (ligangty) [inline src/main/java/org/commonjava/indy/service/tracking/model/AccessChannel.java:35]: Why is this needed? Is this causing some errors?

Comment (hjmodi) [reply src/main/java/org/commonjava/indy/service/tracking/model/AccessChannel.java:35]: yes, old records have an access channel set to this and when constructing those records it throws an error so we need this.

Comment (ligangty) [reply src/main/java/org/commonjava/indy/service/tracking/model/AccessChannel.java:35]: Ok. Let's leave it here.

**Review by ligangty — APPROVED** [review]
LGTM.
